drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -843,12 +1031,13 @@ int stm32_clock_control_init(const struct device *dev)
 	defined(CONFIG_SOC_STM32H7B0XX) || defined(CONFIG_SOC_STM32H7B0XXQ) || \
 	defined(CONFIG_SOC_STM32H7B3XX) || defined(CONFIG_SOC_STM32H7B3XXQ)
 	LL_AHB2_GRP1_EnableClock(LL_AHB2_GRP1_PERIPH_HSEM);
-#else
-	LL_AHB4_GRP1_EnableClock(LL_AHB4_GRP1_PERIPH_HSEM);
-#endif
```

Comment:
Changes impacting H7 series should be made at least in a dedicated commit, ideally in a dedicated PR

Reply:
remove change related to non-stm32h7rs